CODEOWNERS

```
@@ -22,6 +22,7 @@
 
 CODEOWNERS @MrAlias @MadVikingGod @pellared @dashpole
 
+bridges/otellogrus/                                                     @open-telemetry/go-approvers @dmathieu
```

```suggestion
bridges/otellogrus/                                                     @open-telemetry/go-approvers @dmathieu @pellared
```

I can also be a codeowner. I used to use `logrus`.